types/Cargo.toml

```diff
@@ -38,7 +43,7 @@ move-binary-format = { workspace = true }
 move-core-types = { workspace = true }
 move-table-extension = { workspace = true }
 move-vm-types = { workspace = true }
-num-bigint = { workspace = true }
+num-bigint = "0.4.5"#{ workspace = true }
```

Comment:
Ehhh... we shouldn't do this.

Reply:
Resolved by commenting out the code that uses this version to read in the circuit